core/commands/bitswap.go

```diff
@@ -74,12 +74,15 @@ Print out all blocks currently on the bitswap wantlist for the local peer.`,
 	},
 	Encoders: cmds.EncoderMap{
 		cmds.Text: cmds.MakeTypedEncoder(func(req *cmds.Request, w io.Writer, out *KeyList) error {
+			enc, err := cmdenv.GetLowLevelCidEncoder(req)
+			if err != nil {
+				return err
+			}
 			// sort the keys first
 			cidutil.Sort(out.Keys)
 			for _, key := range out.Keys {
-				fmt.Fprintln(w, key)
+				fmt.Fprintln(w, enc.Encode(key))
```

Comment:
Not a strong opinion, but maybe we should do `cmdenv.Encode(req, cid)`?

Reply:
I don't see the benefit and that will also be ineffecnt as it will require reparsing the command line arguments each time.

Reply:
Not to mention that the function would need to return an error, which will make it less convenient to use then the method.